consensus/types/src/chain_spec.rs

```diff
@@ -1141,6 +1254,96 @@ const fn default_attestation_subnet_prefix_bits() -> u8 {
     6
 }
 
+const fn default_max_request_blocks() -> u64 {
+    1024
+}
+
+const fn default_max_request_blocks_deneb() -> u64 {
+    128
+}
+
+const fn default_max_request_blob_sidecars() -> u64 {
+    768
+}
+
+const fn default_min_epochs_for_blob_sidecars_requests() -> u64 {
+    4096
+}
+
+const fn default_blob_sidecar_subnet_count() -> u64 {
+    6
+}
+
+const fn default_epochs_per_subnet_subscription() -> u64 {
+    256
+}
+
+const fn default_attestation_propagation_slot_range() -> u64 {
+    32
+}
+
+const fn default_maximum_gossip_clock_disparity_millis() -> u64 {
+    500
+}
+
+fn min_blocks_by_root_request_common(max_request_blocks: u64) -> usize {
+    let max_request_blocks = max_request_blocks as usize;
+    RuntimeVariableList::<Hash256>::from_vec(Vec::<Hash256>::new(), max_request_blocks)
+        .as_ssz_bytes()
+        .len()
```

Comment:
This function is probably not necessary as the the maximum length parameter isn't used in SSZ serialization and is always going to return 0 for any `max_request_blocks` provided.

Reply:
Oh true! I've removed this method, and the blobs equivalent. And also removed them from `ChainSpec`. Great call, removes a decent amount of complexity!

Reply:
https://github.com/sigp/lighthouse/pull/4841/commits/47410a60e9c0c3d6d12829c45bc9b05c3478e11b